src/System.Net.HttpListener/src/System/Net/Managed/HttpStreamAsyncResult.cs

```diff
@@ -95,10 +95,7 @@ public WaitHandle AsyncWaitHandle
             }
         }
 
-        public bool CompletedSynchronously
-        {
-            get { return (_synchRead == _count); }
```

Comment:
Can we rename _synchRead as well?  It's really now just the number of bytes read, right?